frame/nomination-pools/src/lib.rs

```diff
@@ -1451,7 +1449,15 @@ pub mod pallet {
 						.unwrap_or_default(),
 					Zero::zero()
 				);
-				T::Currency::make_free_balance_be(&bonded_pool.reward_account(), Zero::zero());
+				let reward_pool_remaining = T::Currency::free_balance(&reward_account);
```

Comment:
a test for this would be good. Probably you can just add an assertion in a test that you destroy a pool to make sure the ed is given back to the depositor.

Reply:
Already done in tests like https://github.com/paritytech/substrate/pull/10694/commits/9a35ad614fe32bbcd21a862b5a89381da6677dc9#diff-37b151044ca42c399ae13d489d8312829c2bf0478e93959b0f34c547a2a0fa5cR2152